README.md

```diff
@@ -64,24 +64,19 @@ Try it on [her website!](https://olivia-ai.org)
 ### Installation 
 #### Login to Github 
 
-To get a personal accsess token from Github go to Setings>Developer settings>Personal access tokens
+To get a personal access token from Github go to Setings>Developer settings>Personal access tokens
```

Comment:
I'd put that in a code bracket

Reply:
@hugolgst  by code bracket what do you mean?

Reply:
```
To get a personal access token from Github go to `Setings>Developer settings>Personal access tokens`
```

Reply:
Ok got it, thank you for the clarification.